nats/errors.py

```diff
@@ -65,8 +69,11 @@ def __str__(self) -> str:
 
 
 class SlowConsumerError(Error):
-    def __init__(self, subject=None, sid=None, sub=None) -> None:
+    def __init__(
```

Comment:
@KMilhan , @wallyqs if that's ok with you, those should be the last changes of this PR. I removed the usage of `Optional` (as well as the import from the `nats.errors` module, because it's not used anywhere else)